WcaOnRails/spec/lib/results_validators/persons_validator_spec.rb

```diff
@@ -154,6 +157,22 @@
                                                 competition: competition1,
                                                 eventId: "333oh")
         res_bad_parenthesis.person.update(name: "Bad Parenthesis Guy（test）")
+        res_lowercase1 = FactoryBot.create(:inbox_result,
+                                           competition: competition1,
+                                           eventId: "333oh")
+        res_lowercase1.person.update(name: "First Middle last")
+        res_lowercase2 = FactoryBot.create(:inbox_result,
+                                           competition: competition1,
+                                           eventId: "333oh")
+        res_lowercase2.person.update(name: "İlis хocavənd V")
+        res_missing_period = FactoryBot.create(:inbox_result,
+                                               competition: competition1,
+                                               eventId: "333oh")
+        res_missing_period.person.update(name: "First A B. Last (最初)")
```

Comment:
lol, that Japanese part actually means "first" (in a row of elements or in a sequence).
If you want to test a typical Japanese generic name, pick "Yamada Taro (山田太郎)".

Reply:
Yeah, my intention was to just add something representing a local name without putting much thought. It might be a good idea to write a bit more sensible test names.